hedera-node/hedera-app/src/test/java/com/hedera/node/app/workflows/ingest/IngestCheckerTest.java

```diff
@@ -237,32 +257,36 @@ public static Stream<Arguments> failureReasons() {
                     Arguments.of(INVALID_ACCOUNT_ID));
         }
 
-        @ParameterizedTest(name = "SignatureCheck fails with error code {0}")
-        @MethodSource("failureReasons")
-        @DisplayName("If the payer signature is invalid, the transaction should be rejected")
-        void payerSignatureFails(ResponseCodeEnum failureReason) throws PreCheckException {
-            doThrow(new PreCheckException(failureReason))
-                    .when(signaturePreparer)
-                    .syncGetPayerSigStatus(any());
-
-            assertThatThrownBy(() -> subject.runAllChecks(state, tx))
-                    .isInstanceOf(PreCheckException.class)
-                    .has(responseCode(failureReason));
-        }
-
-        @Test
-        @DisplayName("If some random exception is thrown from checking signatures, the exception is bubbled up")
-        void randomException() throws PreCheckException {
-            // Given an IngestChecker that will throw a RuntimeException from checkPayerSignature
-            doThrow(new RuntimeException("checkPayerSignature exception"))
-                    .when(signaturePreparer)
-                    .syncGetPayerSigStatus(any());
-
-            // When the transaction is submitted, then the exception is bubbled up
-            assertThatThrownBy(() -> subject.runAllChecks(state, tx))
-                    .isInstanceOf(RuntimeException.class)
-                    .hasMessageContaining("checkPayerSignature exception");
-        }
+        //        @ParameterizedTest(name = "SignatureCheck fails with error code {0}")
```

Comment:
Should these be uncommented ?

Reply:
Oh thanks! I will get those working again.